lib/Doctrine/DBAL/Event/SchemaCreateTableEventArgs.php

```diff
@@ -49,7 +48,7 @@ public function getTable()
     }
 
     /**
-     * @return Column[]
+     * @return mixed[][]
```

Comment:
This change kinda hurts... Do we have a way to upcast the `$columns` to `Column[]`?

Reply:
It never returned `Column[]`, there shouldn't be existing clients which would rely on that.